### ISSUE TITLE
Changing time zone changes google calendar entries

### ISSUE THREAD

**Issue by salzrat** [phw198/OutlookGoogleCalendarSync#579]
**OGCS Version**: v2.7.1.0  
**Installed or Portable**: Installed

**Problem Description**  
When moving to another timezone, all entries in google calendar get incorrectly updated.

**Steps to Reproduce the Issue**  
I recently had a trip to a city 9 hours in the west. It seems that during an automatic sync during that trip, all entries got moved 9 hours back in the google calendar. I had applied the second time zone and swap timezone commands in outlook 2010 to temporarily change my windows 7 timezone. When I came back, the entries weren't updated back.

**Comment by phw198** [comment]
Can you replicate the same behavior in the latest alpha release?

To change previously synced timezones back, perform a full sync by pressing and holding `Shift` while clicking `Sync`.

**Comment by salzrat** [comment]
The shift-click didn't help because all events were set to my current timezone both in gcal and outlook - it was just the time that was incorrect.

I updated to 2.7.6 and couldn't replicate the issue (set timezone 9 hours west, sync, set time zone 9 hours back, sync). I restricted sync range to one event, and this stayed perfect. 

Maybe a sync was in progress when I switched time zones some time ago, so it got conflicting information? I don't know how it happened, but I had to change the time for nearly 100 events.
